### PR TITLE
fix: Improved source code formatting at data_structures/linked_list/stack_using_linked_lists.c

### PR DESCRIPTION
**Pull request by Suraj-Patro**
#### Description of Change
Improve source code formatting at data_structures/linked_list/stack_using_linked_lists.c

#### Checklist

- [x] Added description of change
- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C/pull/768"></a>

### CODE REVIEW

File: data_structures/linked_list/stack_using_linked_lists.c [hunk 1]
```diff
@@ -16,8 +16,8 @@ int main()
     printf("\t****stack using linked list****\n");
     while (x != 4)
     {
-        printf("enter your choice");
-        printf("\n1.push\n2.pop\n3.display\n4.exit\n");
+        printf("\n1. Push\n2. Pop\n3. Display\n4. Exit\n");
+        printf("Enter your choice: ");
```
Comment (Suraj-Patro) [inline data_structures/linked_list/stack_using_linked_lists.c:20]: Improve source code formatting

File: data_structures/linked_list/stack_using_linked_lists.c [hunk 2]
```diff
@@ -41,42 +41,42 @@ void push(struct node *p)
     int item;
     struct node *temp;
     temp = (struct node *)malloc(sizeof(struct node));
-    printf("enter element to be inserted\n");
+    printf("\nEnter element to be inserted: ");
```
Comment (Suraj-Patro) [inline data_structures/linked_list/stack_using_linked_lists.c:44]: Improve source code formatting

File: data_structures/linked_list/stack_using_linked_lists.c [hunk 3]
```diff
@@ -41,42 +41,42 @@ void push(struct node *p)
...
     scanf("%d", &item);
     temp->info = item;
 
     temp->link = top;
     top = temp;
 
-    printf("inserted succesfully\n");
+    printf("Inserted succesfully.\n");
```
Comment (Suraj-Patro) [inline data_structures/linked_list/stack_using_linked_lists.c:51]: Improve source code formatting

File: data_structures/linked_list/stack_using_linked_lists.c [hunk 4]
```diff
@@ -41,42 +41,42 @@ void push(struct node *p)
...
 }
 void pop(struct node *p)
 {
     int item;
     struct node *temp;
 
     if (top == NULL)
-        printf("stack is empty\n");
+        printf("\nStack is empty.\n");
```
Comment (Suraj-Patro) [inline data_structures/linked_list/stack_using_linked_lists.c:59]: Improve source code formatting

File: data_structures/linked_list/stack_using_linked_lists.c [hunk 5]
```diff
@@ -41,42 +41,42 @@ void push(struct node *p)
...
     else
     {
         item = top->info;
         temp = top;
         top = top->link;
         free(temp);
-        printf("Element popped is%d\n", item);
+        printf("\nElement popped is %d.\n", item);
```
Comment (Suraj-Patro) [inline data_structures/linked_list/stack_using_linked_lists.c:66]: Improve source code formatting

File: data_structures/linked_list/stack_using_linked_lists.c [hunk 6]
```diff
@@ -41,42 +41,42 @@ void push(struct node *p)
...
     }
 }
 
 void display(struct node *p)
 {
     if (top == NULL)
-        printf("stack is empty\n");
+        printf("\nStack is empty.\n");
```
Comment (Suraj-Patro) [inline data_structures/linked_list/stack_using_linked_lists.c:73]: Improve source code formatting

File: data_structures/linked_list/stack_using_linked_lists.c [hunk 7]
```diff
@@ -41,42 +41,42 @@ void push(struct node *p)
...
     else
     {
-        printf("Elements in the stack are\n");
+        printf("\nElements in the stack are:\n");
```
Comment (Suraj-Patro) [inline data_structures/linked_list/stack_using_linked_lists.c:76]: Improve source code formatting

**Review by hemanth-kotagiri — APPROVED** [review]
LGTM.
@Panquesito7, the code has been properly formatted and I approve these changes.

**Review by Panquesito7 — APPROVED** [review]
Good work on updating the code. Thanks for your contribution! 😄 👍

**Pull request merged**